readme.md

```diff
@@ -435,6 +435,7 @@ const g = x => x * 2
 
 ## Pointed Functor
 An object with an `of` function that puts _any_ single value into it.
+The `of` function must satisfy the so-called [laws of natural tranformations](https://en.wikipedia.org/wiki/Natural_transformation)
```

Comment:
do not like wikipedia as a reference. Wikipedia is good, when there is nothing else or when you looking for some keywords. But definitions and explanations are poor